src/odemis/gui/cont/acquisition.py

```diff
@@ -558,6 +558,11 @@ def _on_acquire(self, _):
             self._acq_future = acqmng.acquire(
                 self._acquiStreams.value, self._tab_data.main.settings_obs)
 
+        # Pause live streams before acquiring
+        for s in self._acquiStreams.value:
+            s.is_active.value = False
+            s.should_update.value = False
```

Comment:
The streambar controller does already provide such function. I think it should be possible to call it like `self._tab.streambar_controller.pauseStreams()`. It works on the `.streams`, but if I understand correctly, `.acquisitionStreams` is a subset of .streams, right? If so, it should be fine.
Could you check if that works?